app/src/main/java/ar/nablab/mimerendero/repositories/RepositoriesModule.kt

```diff
@@ -0,0 +1,16 @@
+package ar.nablab.mimerendero.repositories
```

Comment:
Is there any way to document the injection strategy with objects scopes included? At least as a verbose coment on this module.